apps/freelance/src/main.tsx

```diff
@@ -17,16 +18,19 @@ const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement,
 );
 root.render(
-  <StrictMode>
-    <Suspense fallback={<div>Loading...</div>}>
-      <Provider store={store}>
-        <ThemeProvider theme={baseTheme}>
-          <BrowserRouter>
-            <GlobalStyle />
-            <App />
-          </BrowserRouter>
-        </ThemeProvider>
-      </Provider>
-    </Suspense>
-  </StrictMode>,
+  <GoogleOAuthProvider clientId="71848933982-oc9dn519u5k0dp3pp0a8h20maaokdap8.apps.googleusercontent.com">
```

Comment:
I think its not secure save such data in the code

Reply:
fixed